### DISCUSSION TITLE
bug: Upload documents to  knowledge collection

### DISCUSSION THREAD

**Discussion by pablomart** [open-webui/open-webui#6028] [General]
# Bug Report

**Important: Before submitting a bug report, please check whether a similar issue or feature request has already been posted in the Issues or Discussions section. It's likely we're already tracking it. In case of uncertainty, initiate a discussion post first. This helps us all to efficiently focus on improving the project.**

**Let's collaborate respectfully. If you bring negativity, please understand our capacity to engage may be limited. If you're open to learning and communicating constructively, we're more than happy to assist you. Remember, Open WebUI is a volunteer-driven project maintained by a single maintainer, supported by our amazing contributors who also manage full-time jobs. We respect your time; please respect ours. If you have an issue, We highly encourage you to submit a pull request or to fork the project. We actively work to prevent contributor burnout to preserve the quality and continuity of Open WebUI.**

## Installation Method

Clone repo from github.
Install Node js
Install javascript dependencies --> npm i
build app -> npm run build
Create virtual env python
Install python dependecies --> pip install -r requeriment.txt
start app -> backend/start_windows.bat

## Environment

- **Open WebUI Version:** v0.3.32
- **Ollama (if applicable):** v0.3.9

- **Operating System:** Windows 10
- **Browser (if applicable):** Chrome Version 128.0.6613.85
**Confirmation:**

- [x] I have read and followed all the instructions provided in the README.md.
- [x] I am on the latest version of both Open WebUI and Ollama.
- [x] I have included the browser console logs.
- [x] I have included the Docker container logs.
- [x] I have provided the exact steps to reproduce the bug in the "Steps to Reproduce" section below.

## Expected Behavior:

I want to upload 30 documents into a knowledge collection and use that collection in a RAG.

## Actual Behavior:

When I upload more than 6 documents in a collection, the application restarts and stops the process.

## Description

**Bug Summary:**

Create a new knowledge collection and upload more than 6 files
## Reproduction Details

**Steps to Reproduce:**

1. Create a new knowledge collection.
2. Upload more than 6 documents within that collection

## Logs and Screenshots

**Browser Console Logs:**

**Docker Container Logs:**
[Include relevant Docker container logs, if applicable]

**Screenshots/Screen Recordings (if applicable):**
![image](https://github.com/user-attachments/assets/22c085b8-54e5-49a2-9cd6-3ae33befde83)

## Additional Information

[Include any additional details that may help in understanding and reproducing the issue. This could include specific configurations, error messages, or anything else relevant to the bug.]

## Note

If the bug report is incomplete or does not follow the provided instructions, it may not be addressed. Please ensure that you have followed the steps outlined in the README.md and troubleshooting.md documents, and provide all necessary information for us to reproduce and address the issue. Thank you!

**Comment by tjbck** [comment]
Unable to reproduce with the latest dev, keep us updated!

**Reply by ProjectInitiative** [reply]
I am able to reproduce this with the latest docker container. I am also using v0.3.32. I get a bunch of duplicate file errors despite the files being unique, and then RAG just doesn't work. I'll have to do more research to provide any valuable debugging info
